### PR TITLE
Better support zh_gb.pgf

### PR DESCRIPTION
**Pull request by sum2012**
Before

After

I don't know that ppsspp doesn't choose best font of "disc0:/PSP_GAME/USRDIR/zh_gb.pgf" of this game,so that I need force to use that font when exist.

### CODE REVIEW

**Comment by sum2012** [comment]
tag @lightfirst

File: Core/HLE/sceFont.cpp [hunk 1]
```diff
@@ -645,9 +645,26 @@ static void __LoadInternalFonts() {
 	if (!pspFileSystem.GetFileInfo(fontPath).exists) {
 		pspFileSystem.MkDir(fontPath);
 	}
+	if (pspFileSystem.GetFileInfo("disc0:/PSP_GAME/USRDIR/zh_gb.pgf").exists) {
+		for (size_t i = 0; i < ARRAY_SIZE(fontRegistry); i++) {
```
Comment (unknownbrackets) [inline Core/HLE/sceFont.cpp]: It seems like this adds the first font only, and then bails out... I'm pretty sure that's not the way the PSP works.  Maybe instead there's a problem in the font matching function?

-[Unknown]

Comment (sum2012) [inline Core/HLE/sceFont.cpp]: This is orignal log:
https://gist.github.com/sum2012/8357a3503e4374e862221e097c72fd36
It seem that the chinese team patched the font.prx (or use scereg ?) to force to use it's custom font.
JPCSP also have this problem although it has scereg.

Comment (unknownbrackets) [inline Core/HLE/sceFont.cpp]: Well, I'm not sure it makes sense to affect ALL games for this one game with a patched font.prx.  Perhaps we could detect the patch somehow?

-[Unknown]

**Comment by sum2012** [comment]
@unknownbrackets Add more check (oldfont.prx) should be safe enough

**Comment by unknownbrackets** [comment]
Okay, I guess that seems okay.

-[Unknown]

**Pull request merged**